maven-plugin/src/test/java/io/confluent/kafka/schemaregistry/maven/derive/schema/DeriveSchemaTest.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright 2022 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.confluent.kafka.schemaregistry.maven.derive.schema;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Testing mergeArrays and mergeRecords functions
+ **/
+public class DeriveSchemaTest {
+
+  private final ObjectMapper mapper = new ObjectMapper();
+  DeriveJsonSchema derive = new DeriveJsonSchema();
+  String emptyArray = "{\"type\":\"array\",\"items\":{}}";
```

Comment:
nit: best practice is to define these also as constants with naming in uppercase for them to be better identifiable